omero/training-server/playbook.yml

```diff
@@ -203,6 +203,26 @@
         group: "omero-server"
         force: yes
 
+    - name: Download the Dataset_Images_To_New_Figure.py script
+      become: yes
+      get_url:
+        url: https://raw.githubusercontent.com/ome/omero-guide-figure/master/scripts/Dataset_Images_To_New_Figure.py
```

Comment:
Main downside of using `master` is that a re-execution of this playbook might lead to the deployment of unwanted changes. Is the purpose to track the development branch?

Reply:
The problem here is the scarcity of releases on the omero-figure-guide repo cc @jburel - I think if I would go for the latest release, then, as this is from 2020, the scripts we are discussing here would not be present at all.

Reply:
see https://github.com/ome/omero-guide-figure/tags

Reply:
Sure, I guess I was thinking you could use a commit hash